cg/io/config.py

```diff
@@ -0,0 +1,35 @@
+"""Module to read or write config files"""
+
+from pathlib import Path
+from typing import Any
+
+
+def read_config(file_path: Path) -> Any:
+    """Read content in a config file"""
+    with open(file_path, "r") as file:
+        return file.read()
+
+
+def write_config(content: Any, file_path: Path) -> None:
+    """Write content to a config file"""
+    with open(file_path, "w") as file:
+        file.write(content)
+
+
+def concat_configs(file_paths: list[Path], target_file: Path, str_content: str = "") -> None:
+    """Concatenate config files and string content"""
+    content = str_content
+    for file_path in file_paths:
+        content = content + read_config(file_path)
+    write_config(content, target_file)
+
+
+def write_config_nextflow_style(content: dict[str, Any]) -> str:
+    """Write content to stream accepted by Nextflow config files with non-quoted booleans and quoted strings."""
+    string = ""
+    for key, value in content.items():
+        if isinstance(value, Path):
+            value: str = value.as_posix()
+        quotes = '"' if type(value) is str else ""
```

Comment:
If you like, we defined some string constants in `cg/constants/symbols.py` to make the code more readable. You could use this one for clarity. It is also a bit hard to read the initial double quote '"', you could consider creating a constant for this str as well
```suggestion
        quotes = '"' if type(value) is str else EMPTY_STRING
```